nucliadb_vectors/src/service/writer.rs

```diff
@@ -398,9 +475,24 @@ impl VectorWriterService {
                 index: Index::open(path)?,
                 indexset: IndexSet::new(indexset)?,
                 channel: config.channel,
+                config: config.clone(),
             })
         }
     }
+
+    fn _get_segment_ids(&self, location: &PathBuf) -> NodeResult<Vec<String>> {
```

Comment:
Im confused by this function and the one without the underscore, when is the second one used?
In rust a name is prefixed with an underscore when it is not used.

Reply:
I've renamed it to hopefully improve readability.

The difference here is that we're trying to get segment ids for a particular vectorset

In order to get all the files that we need to sync, we need to iterate through all the vector sets